snap/snapcraft.yaml

```diff
@@ -1,52 +1,145 @@
-name: spark-k8s # you probably want to 'snapcraft register <name>'
+name: spark-client
 base: core20
-version: '0.1'
-summary: Client side snap to submit spark jobs to kubernetes.
+version: '3.3.0'
+summary: Client side scripts to submit Spark jobs to a cluster.
 description: |
-  Spark is now available on Kubernetes and needs a client to submit and 
-  interact with jobs. This snap packages the client binaries for the same.
+  The spark-client snap includes the scripts spark-submit, spark-shell, pyspark and other tools for managing Apache Spark jobs.
 
-grade: devel
+grade: stable
 confinement: strict
 
+plugs:
+  dot-scala-history:
+    interface: personal-files
+    write:
+    - $HOME/.scala_history
```

Comment:
In the `conf/spark-default.conf` configuration, we refer to $SNAP_USER_COMMON/.scala_history. The two should be aligned?